src/site/content/en/blog/mailru-cwv/index.md

```diff
@@ -0,0 +1,290 @@
+
+In addition to bundle size reduction and positive effects on CWV, code splitting improves the developer experience, as a 3.5% share of our users use legacy browsers. Since this share of users is on a downward trend, our team decided to rebuild the process of implementation of new features in the widget. This allows developers to use the latest browser APIs despite the bloat that polyfills introduce for users of legacy browsers.
+
+## Results
+
+After the optimization effort, we observed the mean values for the week of 24–30 May 2021 in our field data:
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th colspan="2" width="40%">Metrics group name</th>
+        <th colspan="3" width="30%">CWV</th>
+        <th colspan="3" width="30%">Other Web Vitals</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td colspan="2">Metric name</td>
+        <td>LCP</td>
+        <td>FID</td>
+        <td>CLS</td>
+        <td>FCP</td>
+        <td>TBT</td>
+        <td>TTI</td>
+      </tr>
+      <tr>
+        <td rowspan="3">Share of users in accordance with CWV thresholds</td>
+        <td>good</td>
+        <td>58% (+6%)</td>
+        <td>93% (+1%)</td>
+        <td>93% (+60%)</td>
+        <td>43% (+8%)</td>
+        <td>49% (+7%)</td>
+        <td>51% (+8%)</td>
+      </tr>
+      <tr>
+        <td>needs-improvement</td>
+        <td>18%</td>
+        <td>4%</td>
+        <td>3%</td>
+        <td>34%</td>
+        <td>17%</td>
+        <td>24%</td>
+      </tr>
+      <tr>
+        <td>poor</td>
+        <td>24%</td>
+        <td>3%</td>
+        <td>4%</td>
+        <td>23%</td>
+        <td>34%</td>
+        <td>25%</td>
+      </tr>
+    </tbody>
+    <caption>CWV metrics for the week of 24–30 March 2021</caption>
+  </table>
+</div>
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/7FIE2St8w6EEowUIqV9K.png", alt="Comparison of Web Vitals before and after (change in 'good' group is shown in brackets).", width="800", height="376" %}
+  <figcaption>Comparison of Web Vitals before and after (change in 'good' group is shown in brackets).</figcaption>
+</figure> 
+
+The graphs below show changes in web page performance metrics values according to the "Platform". Note the two important dates on the graphs:
+ 
+- 23 March 2021: the release of iteration with the last web page blocks migrated to Svelte;
+- 19 April 2021: the release of iteration with returned SSR and layout modified to correct CLS CWV.
+
+The decrease in values from May 1 to May 10 is due to May holidays in Russia.
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/8wq9d0uj3EJp5vK6kI49.png", alt="LCP CWV graph in 'Platform': 16 March to 1 June 2021.", width="800", height="344" %}
```

Comment:
These graphs are pretty hard to describe in alt text :\ I think we should keep them, but hopefully the above table is helpful context.